doc/architecture-decisions/0018-create-policy-server.md

```diff
@@ -0,0 +1,52 @@
+# 17. Create a Policy Server to Manage Network Policy
+
+Date: 2020-10-27
+
+## Status
+
+Accepted
+
+## Context
+
+Well implemented 12 factor apps typically include backend services who never
```

Comment:
should we include the link for https://12factor.net/ ?

Reply:
I wasn't sure I wanted to reference 12 factor at all actually but I can't think of a more general way to phrase it. Maybe "service oriented apps" or something?

Reply:
that make sense yes or cloud native :) I just found 12 factor app being very attached to that manifest :)

Reply:
Boom done thanks